modules/azure/client_factory_test.go

```diff
@@ -0,0 +1,106 @@
+// /*
+
+// This file contains unit tests for the client factory implementation(s).
+
+// */
```

Comment:
NIT: what's with the strange comment syntax?

Reply:
Double-commenting resulting from VSCode. Fixed and removed multi-line comment markers.